src/stream.c

```diff
@@ -1230,7 +1247,7 @@ uvc_error_t uvc_stream_stop(uvc_stream_handle_t *strmh) {
       int res = libusb_cancel_transfer(strmh->transfers[i]);
       if(res < 0 && res != LIBUSB_ERROR_NOT_FOUND ) {
         free(strmh->transfers[i]->buffer);
-        libusb_free_transfer(strmh->transfers[i]);
+        /* transfers are freed in callback */
```

Comment:
This is the only change I'm not so sure about -- could you explain a bit further?

Reply:
When libusb_cancel_transfer is called, callback `void LIBUSB_CALL _uvc_stream_callback(struct libusb_transfer *transfer)` receives series of calls  with `transfer->status` set to `LIBUSB_TRANSFER_CANCELLED` and frees transfers by calling 
```
free(transfer->buffer);
libusb_free_transfer(transfer);
```
function `uvc_stream_stop()` tries to free memory second time what causes crash
If we want to free transfers in `uvc_stream_stop()` we have to wait until callback is called for all transfers